### ISSUE TITLE
Improvements in VISpipeline tutorial

### ISSUE THREAD

**Issue by MjBordalo** [danforthcenter/plantcv#135]
I am following the VIS pipeline tutorial (http://plantcv.readthedocs.io/en/latest/vis_tutorial/) and found some minor flaws.

### 1. Extra agruments

-r results.txt -w are undefined arguments

### 2. Problem with debug argument

This could probably be a problem of mine. But the flag 'print' of the -D argument is not working for me i have to overwrite it in the code if i want the debug mode to print my images

### 3. Output directory
Not sure why, i hv specified the output directory but it is printing to the main project folder.

### 4. Sample image different resolution
I belive the image you make available to run this tutorial is different the one you used. because the ROI rectangle definition does not match.
### 5. color_img not defined
throughout the code i dont think `color_img` was actually created. Got this error:
`NameError: global name 'color_img' is not defined `

**Comment by MjBordalo** [comment]
regarding  1. I see that this arguments will actually be usefull further in the image analysis part. Tought they should be defined in the options method i guess.
I also got this error regarding an image that was not defined:
'NameError: global name 'color_img' is not defined '

**Comment by nfahlgren** [comment]
Yep, you are right, both -r and -w should be defined in the `options` function if the script is to be used with `plantcv-pipeline.py`. Here's what we normally use:

```
def options():
    parser = argparse.ArgumentParser(description="Imaging processing with opencv")
    parser.add_argument("-i", "--image", help="Input image file.", required=True)
    parser.add_argument("-o", "--outdir", help="Output directory for image files.", required=False)
    parser.add_argument("-r","--result", help="result file.", required= False )
    parser.add_argument("-w","--writeimg", help="write out images.", default=False, action="store_true")
    parser.add_argument("-D", "--debug", help="Turn on debug, prints intermediate images.", action="store_true")
    args = parser.parse_args()
    return args
```

**Comment by nfahlgren** [comment]
The debug option is also incorrect in the tutorial. Rather than being a boolean, it should have a default value of `None` and can be set to either "print" or "plot" by the user.

**Comment by nfahlgren** [comment]
The output directory will store images output by the analyze functions when output images are enabled. Images output by the debug option are always written to the current working directory.

**Comment by nfahlgren** [comment]
You are correct about the image resolutions in the documentation. These are intentionally resized to be small so that they load quickly and reduce the file size of the documentation. It would be a good idea if we posted all the documentation sample images (or at least the tutorial images) on Figshare or Zenodo so that someone could test with them.

**Comment by nfahlgren** [comment]
Instead of `color_img`, in the example given, it should be `norm_slize`.

**Comment by jshoyer** [comment]
I still think (per #30, and similar to recently closed issue #110) that the documentation would benefit from some gentler introductory examples. The vis pipeline is somewhat long and complicated, whereas [ImageHarvest example #2](http://cropstressgenomics.org/data/html/ex_script_camera2.html) makes thresholding look easy.
I have not gotten around to writing new walkthroughs mainly because I think they would be easy to maintain as Jupyter notebooks and I am not sure if any of the options for publicly hosting such notebooks are great.

**Comment by maliagehan** [comment]
Updates have been made to correct the problems with the VIS and VIS/NIR tutorial. Please let us know if you find any other problems. Thank you!
